server/src/entities/Game.ts

```diff
@@ -1,20 +1,28 @@
 import { Entity, PrimaryKey, Property } from "mikro-orm";
 import { Field, ObjectType } from "type-graphql";
+import { User } from "./User";
 
 @ObjectType()
 @Entity()
-export class User {
+export class Game {
```

Comment:
The current implementation doesn't seem to allow for multiple players in one game.

There should be two tables, Game and GameUser. Game just stores info for the game: id, start and end times. GameUser should have game id and user id as foreign (and primary) keys, and contain the user's score, number of questions answered correctly, and total number of questions answered.

Reply:
i created sublogin to allow other user to login and get their username if want gameUser can create.